scripts/build_posix.sh

```diff
@@ -46,6 +46,7 @@ fpm_build()
 	fpm -s dir \
 	    -f \
 	    -t $1 \
+		--description "Data Version Control - datasets, models, and experiments versioning for ML or data science projects"
```

Comment:
let's define DESC along others (see the top of this file) and use it here something like `--description 
$DESC`.

Reply:
Also, you've forgot `\` at the end of the line.